package.json

```diff
@@ -138,8 +138,8 @@
     "vscode-builtin-vb": "https://github.com/theia-ide/vscode-builtin-extensions/releases/download/v1.39.1-prel/vb-1.39.1-prel.vsix",
     "vscode-builtin-icon-theme-seti": "https://github.com/theia-ide/vscode-builtin-extensions/releases/download/v1.39.1-prel/vscode-theme-seti-1.39.1-prel.vsix",
     "vscode-builtin-xml": "https://github.com/theia-ide/vscode-builtin-extensions/releases/download/v1.39.1-prel/xml-1.39.1-prel.vsix",
-    "vscode-builtin-yaml": "https://github.com/theia-ide/vscode-builtin-extensions/releases/download/v1.39.1-prel/yaml-1.39.1-prel.vsix",
```

Comment:
vscode-yaml cannot work without this extension

Reply:
> cannot work 
I can see the CA and the validation. Highlighting also works in the editor.

If I have both, I can see errors and warnings because of colliding preference keys.

Reply:
ok, indeed it overrides default language: https://github.com/redhat-developer/vscode-yaml/blob/5e339a44a46aceb48ed2aedabc3146ab5db63559/package.json#L43-L70